google-cloud-document_ai/samples/README.md

```diff
@@ -0,0 +1,41 @@
+# Google Document AI API Samples
+
+Document AI is a document understanding platform that takes unstructured data from documents and transforms it into structured data, making it easier to understand, analyze, and consume.
+
+## Description
+
+These samples show how to use the [Google Document AI API](https://cloud.google.com/document-ai/).
+
+## Build and Run
```

Comment:
since it's "Run" shouldn't it have instruction to run the test, e.g. `bundle exec rake test` or the sample, e.g. ` SampleLoader.load("quickstart.rb").run ... ` somewhere?

Reply:
Added `bundle exec rake test`